sdk/cosmos/azure-cosmos/README.md

```diff
@@ -76,6 +76,33 @@ KEY = os.environ['ACCOUNT_KEY']
 client = CosmosClient(URL, credential=KEY)
 ```
 
+### AAD Authentication
+
+You can also authenticate a client utilizing your service principal's AAD credentials and the azure identity package. 
+You can directly pass in the credentials information to ClientSecretCrednetial, or use the DefaultAzureCredential:
+```Python
+from azure.cosmos import CosmosClient
+from azure.identity import ClientSecretCredential, DefaultAzureCredential
+
+import os
+url = os.environ['ACCOUNT_URI']
+tenant_id = os.environ['TENANT_ID']
+client_id = os.environ['CLIENT_ID']
+client_secret = os.environ['CLIENT_SECRET']
+
+# Using ClientSecretCredential
+aad_credentials = ClientSecretCredential(
+    tenant_id=tenant_id,
+    client_id=client_id,
+    client_secret=client_secret)
+
+# Using DefaultAzureCredential (recommended)
+aad_credentials = DefaultAzureCredential()
+
+client = CosmosClient(url, aad_credentials)
+```
+More information on allowed operations for AAD authenticated clients: [RBAC Permission Model](https://aka.ms/cosmos-native-rbac)
```

Comment:
This is the wrong link; it should be https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-setup-rbac
We should also mention explicitly that the AAD token must have permissions to:
Microsoft.DocumentDB/databaseAccounts/readMetadata

Reply:
This is the link to the permission model showing the allowed operations based on permissions granted to the managed identity, I'll add the link you posted to help people with setup too though, that's good information to put there for sure.

Reply:
That previous link is different, the AAD token obtained though just RBAC setting will not work with this implementation.